javascript/javascript1/week2/homework-week2/event-application.js

```diff
@@ -0,0 +1,8 @@
+const week = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday'];
+
+function getEventWeekday(daysToEvent){
+    return `Today is ${new Date()} and the event is on ${ week[(new Date().getDay() + daysToEvent) % week.length] }`
+}
+
+console.log(getEventWeekday(1));
```

Comment:
This Function has a  bug , like 1 returns two days later. I mean basically adds a day extra. please fix it.